addon/-private/execution_context/helpers.js

```diff
@@ -36,3 +36,34 @@ export function fillElement(selection, content, { selector, pageObjectNode, page
     $selection.val(content);
   }
 }
+
+/**
+ * @private
+ *
+ * Given an element, asserts that element is focusable/blurable
+ *
+ * @param {Element} element - the element to check
+ */
+export function assertFocusable(selection, { selector, pageObjectNode, pageObjectKey }) {
+  let $selection = $(selection);
+
+  let error;
+
+  if ($selection.is(':hidden')) {
+    error = 'hidden';
+  } else if ($selection.is(':disabled')) {
```

Comment:
Let's add an additional block for `[contenteditable="false"]` so that `[contenteditable]` on line 56 excludes elements that have the `contenteditable` attr but which are not actually editable.